docs/search_relevance/query_design.md

```diff
@@ -28,11 +28,68 @@ Each query has:
 
 ## Intentions & Expectations
 
+### Phrase matching
+
+| Status | Query name | Ranking evaluation test |
+| :--- | :--- | :--- |
+| TODO | PhraseMatchQuery | TBD |
+
+#### Intentions
+
+Searching for an exact, ordered set of tokens, in quotation marks
+
+#### Data features
+
+* `data.title`
+* `data.alternativeTitle`
+* `physicalDescription`
+* `subjects.label`
+* `genres.label`
+* `contributors.label`
+* `description`
+
+#### Expectations
+
+* Searching for a phrase in quotation marks should match works containing that exact set of tokens, in the same order
+* Works matching individual tokens in the phrase should not be matched by this query
+
+#### Examples
+
+* `The ocean as a health resort : a practical handbook of the sea for the use of tourists and health-seekers`- [https://wellcomecollection.org/works/uxxaqdkg](https://wellcomecollection.org/works/uxxaqdkg) should be the only result returned by this query
```

Comment:
Are you sure about it being the only query? Is this then saying if we match a phrase, we don't use other, more loose queries? 

Or is there an expectation that you point that phrase out? Or let the person know that this is highly relevant?

Reply:
> by this query

Makes this obvious.